homeassistant/components/lock/kiwi.py

```diff
@@ -0,0 +1,166 @@
+"""
+Support for the KIWI.KI lock platform.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/lock.kiwi/
+"""
+import logging
+import datetime
+import requests
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.lock import (LockDevice, PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_PASSWORD, CONF_USERNAME, ATTR_ID, ATTR_LONGITUDE, ATTR_LATITUDE)
+
+REQUIREMENTS = ['python-dateutil==2.6.1']
```

Comment:
Use `homeassistant.util.dt` if you have to deal with time/date objects.

Reply:
ah okay - didn't know about that - that would be nice because It would eliminate the `REQUIREMENTS`